test/NuGet.Core.FuncTests/NuGet.Packaging.CrossVerify.Generate.Test/GenerateSignedPackages.cs

```diff
@@ -361,8 +364,43 @@ public async Task PreGenerateSignedPackages_AuthorSigned_Timestamped_RepositoryC
 
             }
         }
+
+#if IS_DESKTOP
+        [Fact]
+        public async Task PreGenerateSignedPackages_AuthorSigned_TimeStampedWithNoSigningCertificateUsage()
+        {
+            // Arrange
+            var folder = TestPackages.Package1.ToString();
+            string packageFolderPath = Path.Combine(_directoryPath, folder);
+            Directory.CreateDirectory(packageFolderPath);
+
+            ISigningTestServer testServer = await _signingTestFixture_Author.GetSigningTestServerAsync();
+            CertificateAuthority rootCa = await _signingTestFixture_Author.GetDefaultTrustedCertificateAuthorityAsync();
+            var options = new TimestampServiceOptions()
+            {
+                SigningCertificateUsage = SigningCertificateUsage.None
+            };
+            TimestampService timestampService = TimestampService.Create(rootCa, options);
+
+            using (testServer.RegisterResponder(timestampService))
+            {
+                var nupkg = new SimpleTestPackageContext();
+
+                string packagePath = Path.Combine(packageFolderPath, "package");
+                Directory.CreateDirectory(packagePath);
+
+                using (var certificate = new X509Certificate2(_signingTestFixture_Author.TrustedTestCertificate.Source.Cert))
+                {
+                    string signedPackagePath = await SignedArchiveTestUtility.AuthorSignPackageAsync(
```

Comment:
`signedPackagePath` is unused.  How about asserting that the expected file was created successfully?
```C#
Assert.True(File.Exists(signedPackagePath));
```